op/model.py

```diff
@@ -302,3 +312,6 @@ def relation_set(self, relation_id, key, value):
 
     def config_get(self):
         return self._run('config-get')
+
+    def is_leader(self):
+        return self._run('is-leader')
```

Comment:
What happens with _run in such a case, where we're using the exit status?  We have two possible cases here: it just returns False, or it errors out. The logic in the other methods above means that either we had a bug, or we're now adding a bug.

Reply:
When fixing this, we need a more realistic test that ensures this logic is doing its job. The fact we can get these wrong with nothing breaking means we can break it again at any point without noticing it.